scripts/grub.cfg

```diff
@@ -16,18 +16,20 @@ if loadfont $prefix/fonts/unicode.pf2 ; then
   set gfxmode=auto
   load_video
   insmod gfxterm
+  insmod serial
   set locale_dir=$prefix/locale
   set lang=en_US
   set gfxpayload=keep
 fi
-terminal_output gfxterm
 
-serial --unit=0 --speed=9600 --word=8 --parity=no --stop=1
+serial --unit=0 --speed=115200 --word=8 --parity=no --stop=1
+terminal_output console serial
+terminal_input console serial
+sleep -i 1
```

Comment:
What do you think of adding that comment here, the one you replied to Ryan, please?

Sounds useful enough so the next person doesn't get lost :D

Reply:
Sure, I have added the comment.

Reply:
Thank you!